src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -424,6 +424,18 @@
                     </Button>
                 </StackPanel>
             </Border>
+            <Border x:Name="TracePointer"
+                    Width="16"
+                    Height="18"
+                    Margin="50,50,0,0"
+                    HorizontalAlignment="Left"
+                    VerticalAlignment="Top"
+                    Visibility="Collapsed">
+                <FontIcon Foreground="Red"
```

Comment:
>Foreground="Red" [](start = 26, length = 16)

The dot was red before, so I made the cursor red...  I tried going with the theme text color instead, but that resulted in something that didn't pop.  It was hard to see where the pointer was at a glance